From 3026b916ce811f64c36e04fcba2c99e308e22e86 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Mon, 4 Mar 2024 13:47:22 +0100
Subject: [PATCH libaio 25/26] build: Use a temporary SONAME bump to avoid
 stomping over upstream ABI

We are using a different SONAME in case upstream wants to see a
different implementation for the newly added symbols. Where we then can
easily revert to the original SONAME. If upstream takes the symbols as
is, then we can simply create a compat symlink.

Origin: vendor
Forwarded: not-needed
Sent-Upstream: not-relevant
Signed-off-by: Guillem Jover <guillem@hadrons.org>
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index e63e706..9e4fecd 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -10,7 +10,7 @@ L_CFLAGS = $(MK_CFLAGS)
 MK_LDFLAGS = $(LDFLAGS)
 ENABLE_SHARED ?= 1
 
-soname=libaio.so.1
+soname=libaio.so.1t64
 minor=0
 micro=2
 libname=$(soname).$(minor).$(micro)
-- 
2.43.0

